Documentation/CRDs/Cluster/network-providers.md

```diff
@@ -418,6 +418,23 @@ spec:
     }'
 ```
 
+## Holder Pod Deprecation
+
+Rook plans to deprecate CSI "holder" pods in Rook v1.16. Clusters with `csi-*plugin-holder-*` pods
```

Comment:
We are planning to remove support for them completely, right? I think "deprecate" means different things to different people, so just to clarify.
```suggestion
Rook plans to remove CSI "holder" pods in Rook v1.16. Clusters with `csi-*plugin-holder-*` pods
```

Reply:
Okay. I changed this throughout all docs